maven-release-plugin/src/it/projects/prepare/regular-multi-module/module-parent/pom.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You've already created a new IT, is this file still required?

Reply:
Removed in https://github.com/apache/maven-release/commit/42eeaf70dafedc759e7bd5421842d2ad687f7deb.